src/appleseed/foundation/utility/unzipper.cpp

```diff
@@ -0,0 +1,217 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
Replace these two lines of copyright notice by a single line:

    // Copyright (c) 2017 Gleb Mishchenko, The appleseedhq Organization

Use this copyright notice for all new files you create.